### DISCUSSION TITLE
It's now been exactly a year since the last release ;-)

### DISCUSSION THREAD

**Discussion by zlite** [NVIDIA-AI-IOT/jetbot#366] [Ideas]
Jetbot team, happy one year anniversary of the[ last release](https://github.com/NVIDIA-AI-IOT/jetbot/releases/tag/v0.4.0) (Jan 15 2020)!  You've made a huge amount of progress (270 commits) since then. Maybe time to roll it up into a new release?

**Comment by jaybdub** [comment]
Hey @zlite ,

Thanks, this is a good observation!  

We actually haven't tagged them, but there have been two releases since 1/15.

http://jetbot.org/master/software_setup/sd_card.html

Apologies for any confusion this may have caused.  I'll see if I can retroactively tag these.

Even still, some changes have been made since, and we might be due for another release tag :) 

Best,
John

**Comment by jaybdub** [comment]
Following up on this.  Tags have been added retroactively for the more recent releases.

https://github.com/NVIDIA-AI-IOT/jetbot/releases

Please let me know if this helps or you have any other suggestions :)

Best,
John

**Comment by zlite** [comment]
Congrats and thanks! Now you have to do it again ASAP for Jetpack 4.5 ;-)

**Comment by jaybdub** [comment]
Hi @zlite,  

The latest release (targeting JetPack 4.5) is now hosted.  The SD card images are hosted here

*  http://jetbot.org/master/software_setup/sd_card.html

* https://github.com/NVIDIA-AI-IOT/jetbot/releases/tag/v0.4.3

The docker image is also published, instructions should be similar.

Please feel free to reach out if you run into any issues!

Best,
John
